frontend/src/welcome/Welcome.jsx

```diff
@@ -39,8 +39,13 @@ const Welcome = () => {
         <>
             <Toast ref={toast}/>
             <div className="flex flex-column justify-content-center align-items-center gap-4 h-full">
-                <img src="monkeyPollLogoWithWhiteText.png" alt="MonkeyPoll Logo"
-                     style={{width: '90%', maxWidth: '1080px', height: 'auto'}}/>
+                <div className="flex justify-content-center">
+                    <img src="monkeypoll-full-white.svg" alt="MonkeyPoll Logo"
+                         style={{width: '50%', maxWidth: '1080px', height: 'auto'}}/>
+                </div>
+                <div className="flex pb-4">
+                    <p style={{fontSize: '1.4rem'}}>Polls made simple.</p>
```

Comment:
Check out this [styling library](https://primeflex.org/fontsize). Its better if we stuck with their constants and only override for specifics such as our colour theme which they don't support. If you add `text-2xl` within the className of parent div, it should get the same result.

Reply:
Yeah good catch